ansible/setup.yml

```diff
@@ -81,9 +81,21 @@
         update_cache: yes
       with_items:
         - build-essential
+        - python-yaml
       tags:
         - dev-tools
 
+    - name: "Create custom fact directory"
+      file:
+        path: "/etc/ansible/facts.d"
+        state: "directory"
+
+    - name: "Install epoch fact file"
+      copy:
+        src: files/epoch.fact
+        dest: /etc/ansible/facts.d/epoch.fact
```

Comment:
This file seems coupling infrastructure with deployed app, though I see it is in /etc so kind of part of infrastructure.

Ok.